package.json

```diff
@@ -17,7 +17,7 @@
     }
   ],
   "scripts": {
-    "test": "ava && xo",
+    "test": "LD_LIBRARY_PATH=./libui/build/out ava && xo",
```

Comment:
I was not able to find a way to remove the LD_LIBRARY_PATH. Seems silly to need this